config/opentelemetry-collector/collector-config.yaml

```diff
@@ -2,7 +2,8 @@ receivers:
   otlp:
     protocols:
       grpc:
-
+    tls:
```

Comment:
Can we revert this?

Reply:
Yes. I was (unsuccessfully) trying to fix the OTEL Collector CI :/